docs/execution-providers/QNN-ExecutionProvider.md

```diff
@@ -461,3 +465,8 @@ sess = ort.InferenceSession(model_path, providers=['QNNExecutionProvider'], prov
 ### Inference example
 
 [Image classification with Mobilenetv2 in CPP using QNN Execution Provider with QNN CPU & HTP Backend](https://github.com/microsoft/onnxruntime-inference-examples/tree/main/c_cxx/QNN_EP/mobilenetv2_classification)
+
+
+## Error handling
+### HTP SubSystem Restart - [SSR](https://docs.qualcomm.com/bundle/publicresource/topics/80-63442-50/htp_backend.html#subsystem-restart-ssr-)
+QNN EP wil return StatusCode::ENGINE_ERROR regarding QNN HTP SSR issue. Uppper level framework/application should recreate Onnxruntime session if this error detected during session run.
```

Comment:
typo:wil
probably better to change to QNN EP returns StatusCode::ENGINE_ERROR ...